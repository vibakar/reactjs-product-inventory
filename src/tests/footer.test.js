import React from 'react';
import { mount } from 'enzyme';
import Footer from '../components/footer';

describe('Footer page snapshot', () => {
    let wrapper;
    let props;
    beforeEach(() => {
        wrapper = mount(<Footer {...props} />);
    });

    test('renders correctly', () => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
