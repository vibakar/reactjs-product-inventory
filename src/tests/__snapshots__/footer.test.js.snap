// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer page snapshot renders correctly 1`] = `
"<Footer>
  <Navbar fixed=\\"bottom\\" expand=\\"lg\\" bg=\\"dark\\" variant=\\"dark\\" collapseOnSelect={false}>
    <nav className=\\"navbar navbar-expand-lg navbar-dark bg-dark fixed-bottom\\">
      <p className=\\"m-auto footerText\\">
        Copyright Â© 2019
      </p>
    </nav>
  </Navbar>
</Footer>"
`;
